### PR TITLE
src: update to standard@14

### PR DESCRIPTION
**Pull request by rvagg**
This mainly undoes all of my nice, consistent array spacing. They fell down on the (erroneous IMO) side of no spaces either side of square brackets, regardless of their use. Unnecessary quoting of array/object properties is also now checked.

### CODE REVIEW

File: lib/process-release.js [hunk 1]
```diff
@@ -1,3 +1,5 @@
+/* eslint-disable node/no-deprecated-api */
```
Comment (rvagg) [inline lib/process-release.js]: `url.resolve()` and `url.parse()` are used in here. I'm not entirely comfortable switching over fully to `URL` yet, it took a little while to stabilise, so maybe not until we want to drop support for Node 8, or until Node starts printing deprecation warnings to stdout.

**Comment by cclauss** [comment]
> undoes all of my nice, consistent array spacing

Are there formatter directives like [Python Black's __# fmt: off__ and __# fmt: on__](https://github.com/psf/black#the-black-code-style) so we can use sparingly to preserve your array artistry?

**Comment by cclauss** [comment]
Conflicts to resolve.

**Comment by rvagg** [comment]
resolved.

> Are there formatter directives like Python Black's # fmt: off and # fmt: on so we can use sparingly to preserve your array artistry?

No, intentionally so. The purpose of `standard` is to provide a uniform standard that everyone can get on board with. We make a pact with each other to let go of most of our syntax preferences and accept `standard` as close-enough to what we like and just roll with it, removing most of the syntax arguing that's taken up so much GitHub comment space over the years. IMO the `[]` decision was short-sighted but I'm already putting up with it in many other projects and just getting on with the important work.

**Comment by cclauss** [comment]
Agreed.  Same philosophy behind the Python Software Foundation’s __Black__...  You can have any color you want as long as it is Black.  ;-)

**Comment by rvagg** [comment]
landed in 3555ff4f

**Pull request closed without merging**